expirationd.lua

```diff
@@ -71,13 +73,15 @@ local function expiration_process(task, tuple)
     end
 end
 
+-- yield for some time
 local function suspend_basic(scan_space, task, len)
     local delay = (task.tuples_per_iteration * task.full_scan_time)
     delay = math.min(delay / len, task.iteration_delay)
     fiber.sleep(delay)
 end
 
 local function suspend(scan_space, task)
+    -- Return the number of tuples in the space
```

Comment:
Why previous added comments are above function and this one below function?

Reply:
Because this comment is not for a function, but for the line below